static/less/variables.less

```diff
@@ -144,4 +144,8 @@
 
     // Fonts
     --mail-font-family: @mail-font-family;
+
+    // Dimensions
+    --hh-fixed-header-height: 130px;
+    --hh-fixed-footer-height: ~"0px"; // Force keeping `px` after CSS compilation (required for `calc()` function)
```

Comment:
https://stackoverflow.com/questions/29042007/dont-compress-0px-in-0-with-less-css-minificator
https://stackoverflow.com/questions/27829919/why-do-i-have-to-specify-a-unit-identifier-e-g-px-when-passing-0-as-the-onl

Reply:
I haven't tested, but hopefully we won't have to do this hack with SCSS (BS5).

Reply:
@marc-farre Thanks, I've tried the CSS compilation and cannot reproduce.

Example Less:

```
➜  /tmp cat temp.less

:root {
    --hh-fixed-header-height: 130px;
    --hh-fixed-footer-height: ~"0px";
    --hh-fixed-footer2-height: 0px;
}

➜  /tmp lessc temp.less

:root {
  --hh-fixed-header-height: 130px;
  --hh-fixed-footer-height: 0px;
  --hh-fixed-footer2-height: 0px;
} 
```

Did I missed something here?

Reply:
@luke- the issue occurs when compressing with the `-x` param: https://docs.humhub.org/docs/develop/build#manual-theme-build

Reply:
@marc-farre Thanks, got it!

What do you think about switching to the CSS Minifier recommended by Less? 

It's quite easy to switch, we just need to additionally install `npm install -g less-plugin-clean-css` and instead of the `-x` option, we can use `--clean-css`.
During a short test, the `px` value was still present.

What do you think? Should we remove the workaround and switch to this methd? We just need to update the doc/grunt?